Userland/Libraries/LibWeb/Crypto/SubtleCrypto.cpp

```diff
@@ -245,4 +344,49 @@ void SubtleCrypto::define_an_algorithm(String op, String algorithm, String type)
     internal_object.set(op, registered_algorithms);
 }
 
+// https://w3c.github.io/webcrypto/#pbkdf2-operations
+JS::ThrowCompletionOr<JS::NonnullGCPtr<CryptoKey>> SubtleCrypto::pbkdf2_import_key([[maybe_unused]] Variant<ByteBuffer, Bindings::JsonWebKey, Empty> key_data, [[maybe_unused]] AlgorithmIdentifier algorithm_parameter, Bindings::KeyFormat format, bool extractable, Vector<Bindings::KeyUsage> key_usages)
+{
+    auto& realm = this->realm();
+
+    // 1. If format is not "raw", throw a NotSupportedError
+    if (format != Bindings::KeyFormat::Raw) {
+        // FIXME: This should be a NotSupportedError
+        return realm.vm().throw_completion<JS::TypeError>(JS::ErrorType::NotImplemented, idl_enum_to_string(format));
```

Comment:
This function should return WebIDL::ExceptionOr, and then you can return WebIDL::NotSupportedError::create(realm, "inavlid format") or w/e.